packages/testcontainers/src/container-runtime/clients/container/podman-container-client.ts

```diff
@@ -1,20 +1,38 @@
-import { Container } from "dockerode";
-import { ExecResult } from "./types";
+import { Container, ExecCreateOptions } from "dockerode";
+import { ExecOptions, ExecResult } from "./types";
 import byline from "byline";
 import { DockerContainerClient } from "./docker-container-client";
 import { execLog, log } from "../../../common";
 
 export class PodmanContainerClient extends DockerContainerClient {
-  override async exec(container: Container, command: string[], opts?: { log: boolean }): Promise<ExecResult> {
-    const chunks: string[] = [];
+  override async exec(
+    container: Container,
+    command: string[],
+    opts?: Partial<ExecOptions>,
+  ): Promise<ExecResult> {
```

Comment:
```suggestion
  override async exec(container: Container, command: string[], opts?:Partial<ExecOptions>): Promise<ExecResult> {
```

Same prettier format problem